### DISCUSSION TITLE
Fatal error: Uncaught Error: Class 'Phalcon\Config' not found (Windows)

### DISCUSSION THREAD

**Discussion by michaltaglewski** [phalcon/cphalcon#15933] [Q&A]
I'm getting error when trying to create new project using devtool. 

`Fatal error: Uncaught Error: Class 'Phalcon\Config' not found...`

Phalcon and Psr extension DLLs seem to be installed correctly (phpinfo display it)

```
phalcon --version

Phalcon DevTools (4.2.0)

Environment:
  OS: Windows NT DESKTOP-VANCEMC 10.0 build 19044 (Windows 10) AMD64
  PHP Version: 7.4.27
  PHP SAPI: cli
  PHP Bin: G:\xampp\php\php.exe
  PHP Extension Dir: C:\php\ext
  PHP Bin Dir: C:\php
  Loaded PHP config: G:\xampp\php\php.ini
Versions:
  Phalcon DevTools Version: 4.2.0
  Phalcon Version: 5.0.0beta3
  AdminLTE Version: 3.0.1
```
DevTools installed globally with composer.
I tried with PHP 8.0 too - same result.

**Comment by Jeckerson** [comment]
You are using devtools that generated code of Phalcon v4, which is not compatible with v5. Check changelog to adapt generated code - https://github.com/phalcon/cphalcon/releases

Also use stubs, to improve classes visibility - `composer require --dev phalcon/devtools:5.0.0-beta.3` (https://github.com/phalcon/ide-stubs/releases)

**Reply by michaltaglewski** [reply]
I downgraded my Phalcon version to 4.1.2 and that helped 100%. Thank you! Do I understand correctly there isn't devtools v5? Ide stubs also good point.

**Comment by lieshanhe** [comment]
somehow,  the namespace has been changed to 

``
Phalcon\Config\Config
``

so, you can use it like this:

``
use Phalcon\Config\Config;
``

**Reply by borisdelev** [reply]
Yes, there is a reason for that. Same- other stuff and namespaces are changes and improved, so you can view them in [Changelog 5.x.x](https://github.com/phalcon/cphalcon/blob/5.0.x/CHANGELOG-5.0.md).
